### ISSUE TITLE
Dose it work with pytorch?

### ISSUE THREAD

**Issue by huyvan00** [czczup/block-shuffle#1]

**Comment by czczup** [comment]
> _No description provided._

Sorry, we only implemented the tensorflow version.
